app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%= @amusement_park.name %></h1>
+<p>Admission Price: $<%= @amusement_park.admission_cost %></p>
+
+<h2>Mechanics Working on Rides:</h2>
+<ul>
+  <% @amusement_park.rides.flat_map(&:mechanics).uniq.each do |mechanic| %>
```

Comment:
This is what I would consider an MVC infraction - `.flat_map` could conceivably be used to display some data, but when we do `.uniq`, that is taking the place of an ActiveRecord query (`.distinct`) that should live as part of a model method.